prog_test.go

```diff
@@ -99,6 +103,74 @@ func TestProgramBenchmark(t *testing.T) {
 	}
 }
 
+func TestProgramTestRunInterrupt(t *testing.T) {
+	testutils.SkipOnOldKernel(t, "5.0", "EINTR from BPF_PROG_TEST_RUN")
+
+	prog := createSocketFilter(t)
+	defer prog.Close()
+
+	var (
+		tgid    = unix.Getpid()
+		tidChan = make(chan int, 1)
```

Comment:
maybe, we should also close `tidChan`?

Reply:
What would the benefit be? It will be garbage collected regardless, no?